<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RaumeDialog</class>
 <widget class="QDialog" name="RaumeDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>589</width>
    <height>531</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Buildings and rooms</string>
  </property>
  <widget class="QToolBox" name="toolBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>561</width>
     <height>491</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="page">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>561</width>
      <height>433</height>
     </rect>
    </property>
    <attribute name="label">
     <string>Buildings</string>
    </attribute>
    <widget class="QTableView" name="tableBuildings">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>521</width>
       <height>331</height>
      </rect>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="pushAdd">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>370</y>
       <width>82</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>New</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>561</width>
      <height>433</height>
     </rect>
    </property>
    <attribute name="label">
     <string>Rooms</string>
    </attribute>
    <widget class="QTableView" name="tableRooms">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>521</width>
       <height>271</height>
      </rect>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineName">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>300</y>
       <width>311</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>290</y>
       <width>71</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Name</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushAddRoom">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>390</y>
       <width>82</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>330</y>
       <width>71</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Building</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBuilding">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>340</y>
       <width>261</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineCapacity">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>380</y>
       <width>91</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>370</y>
       <width>71</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Capacity</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
